external/fv3fit/tests/sklearn/test_transformer.py

```diff
@@ -0,0 +1,96 @@
+from fv3fit.sklearn.transformer import SkTransformer
+import numpy as np
+import os
+from sklearn.preprocessing import StandardScaler
+
+
+class DummyTransformer:
+    def __init__(self, latent_dim, output_dim, encoded_constant, decoded_constant):
+        self.latent_dim = latent_dim
+        self.output_dim = output_dim
+        self.encoded_constant = encoded_constant
+        self.decoded_constant = decoded_constant
+
+    def transform(self, X):
+        return np.ones(self.latent_dim) + self.encoded_constant
```

Comment:
To try and test a true roundtrip, you could slot the input data into a larger array and then select it out on the inverse.  That would allow for an easier to understand test below.